aries-site/src/pages/components/textarea.js

```diff
@@ -17,19 +22,100 @@ const TextArea = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        <Example
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/textarea/TextAreaExample.js"
+          designer="https://designer.grommet.io/textarea?id=HPE-design-system-taylor-seamans-hpe-com&mode=edit"
+          docs="https://v2.grommet.io/textarea?theme=hpe#props"
+          figma="https://www.figma.com/file/evH1LQewFPc8Y2fteywMxM/HPE-Text-Area-Component"
+        >
+          <TextAreaExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          Use TextArea for multi-line input such as notes or comments. TextArea
+          is used frequently within the context of a form, but there may be
+          cases where it would exist independently. Placeholder text can be used
+          to guide the user towards how to start their entry.
```

Comment:
This is a nice intro. I almost like the first sentence here as a possible replacement for the page.description.